docs/output.md

```diff
@@ -389,6 +389,8 @@ The main taxonomic classification file from Kraken2 is the `_combined_reports.tx
 
 You will only receive the `.fastq` and `*classifiedreads.txt` file if you supply `--kraken2_save_reads` and/or `--kraken2_save_readclassifications` parameters to the pipeline.
 
+When running Kraken2 together with bracken, you can use the `--bracken_save_intermediatekraken2` to save the intermediate/upstream files generated during the bracken run.
```

Comment:
```suggestion
When running Bracken, you will only get the 'intermediate' Kraken2 report files in this directory if you supply `--bracken_save_intermediatekraken2` to the run.
```

Reply:
Please also add  on lines 379 and 383 above that the Bracken files only with also with the flag too